### PR TITLE
[tests-only] Allow setting individual quotas per group

### PR DESCRIPTION
**Pull request by kobergj**
Needs reva PR https://github.com/cs3org/reva/pull/3189 and https://github.com/cs3org/reva/pull/3242 but can be merged independently

### CODE REVIEW

File: services/gateway/pkg/config/config.go [hunk 1]
```diff
@@ -44,6 +44,9 @@ type Config struct {
 
 	StorageRegistry StorageRegistry `yaml:"storage_registry"` //TODO: should we even support switching this?
 
+	DefaultPersonalQuota string `yaml:"default_personal_quota" env:"PERSONAL_SPACES_DEFAULT_QUOTA" desc:"The default quota for personal spaces in bytes. Usable common abbreviations: [KB, KiB, GB, GiB, TB, TiB, PB, PiB, EB, EiB], example: 1GB. 0 means unlimited and is the default."`
+	GroupQuotaFile       string `yaml:"group_quota_file" env:"GROUP_QUOTA_FILE" desc:"The path to the file containing the group quotas. The file needs to be json in the form groupname:quota. Note: If set and the target is not json or the entries are not parseable (see default_personal_quota) the service will refuse to start."`
```
Comment (kobergj) [inline services/gateway/pkg/config/config.go:48]: @mmattel it is only pointing to experimental, but could you still have a look if the description is sufficient? Note: This is only a temporary workaround so this envvar will disappear before it lands in master.

File: services/proxy/pkg/middleware/create_home.go [hunk 2]
```diff
@@ -65,6 +46,28 @@ func (m createHome) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	//ctx := tokenpkg.ContextSetToken(r.Context(), token)
```
Comment (fschade) [inline services/proxy/pkg/middleware/create_home.go:46]: leftover?

**Review by fschade — APPROVED** [review]
one comment, not sure if leftover or intended, expect that LGTM

File: services/proxy/pkg/middleware/create_home.go (hunk 2 above)
Comment (kobergj) [reply services/proxy/pkg/middleware/create_home.go]: Was there before :stuck_out_tongue: But fixed it anyways...

**Pull request merged**